twill-api/src/main/java/org/apache/twill/api/TwillController.java

```diff
@@ -89,4 +90,44 @@
    * @return A {@link Future} that will be completed when the restart operation has been done.
    */
   Future<String> restartInstances(String runnable, int instanceId, int... moreInstanceIds);
+
```

Comment:
By docs on https://github.com/apache/twill/blob/master/twill-core/src/main/java/org/apache/twill/internal/AbstractZKServiceController.java#L107, that means the message was successfully processed.